### PR TITLE
changes to africa map were taken out 

### PR DESCRIPTION
**Pull request by lynnnakiganda**

### CODE REVIEW

File: css/legend_info.css [hunk 1]
```diff
@@ -27,5 +27,22 @@
     background-color: #4e4e4e;
     box-shadow: 0 0 15px rgba(0,0,0,0.2);
     border-radius: 10px;
-    
+
+}
+
+.leaflet-control-layers-expanded {
```
Comment (RonKbS) [inline css/legend_info.css:33]: Hy @lynnnakiganda , can you remove these css changes, please. 
I do not see them being requested for in the change sheet and I personally prefer the sidepanel with the shadows.

File: js/ug_onEachFeature_fns.js [hunk 2]
```diff
@@ -171,10 +171,9 @@ function icu_OEF(layer) {
 }
 
 function water_points_OEF(layer) {
-  layer.bindPopup('<strong>Name:</strong> ' + layer.feature.properties.name + '<br>' + '<strong>District:</strong> ' +
-      layer.feature.properties.addr_district + '<br>' + '<strong>SubCounty:</strong> ' +
-      layer.feature.properties.addr_subcounty + '<br>' + '<strong>Parish:</strong> ' +
-      layer.feature.properties.addr_parish + '<br>' + '<strong>Water Source Coverage (%):</strong> ' +
+  layer.bindPopup('<strong>District:</strong> ' +
+      layer.feature.properties.districts1_DName2016 + '<br>' + '<strong>Total Population:</strong> ' +
```
Comment (RonKbS) [inline js/ug_onEachFeature_fns.js:175]: Hi @lynnnakiganda ,

These fields in the districts data do not match `Total Population` and `Water Source Coverage`

Comment (lynnnakiganda) [reply js/ug_onEachFeature_fns.js:175]: dont match how

Comment (RonKbS) [reply js/ug_onEachFeature_fns.js:175]: `districts1_DName2016` is the name of a district but `Total Population` should be a value if I'm not mistaken...

Comment (lynnnakiganda) [reply js/ug_onEachFeature_fns.js:175]: TotalPopn thats total population and water1_water_percentage is water coverage

Comment (RonKbS) [reply js/ug_onEachFeature_fns.js:175]: That is not what is written in that section...

**Pull request closed without merging**